scst-store/deployment-details.hbs.md

```diff
@@ -51,37 +86,15 @@ Where `PASSWORD-0123` is the same password used between deployments.
 
```

Comment:
It feels weird to me that this is the only section that mentions `metadata_store` if installing via tap values. I wonder if there should be another section that mentions that all configurations can be nested inside of `metadata_store`.

Reply:
hm I think I'll remove this `metadata_store`, and just mention at the beginning that they are all nested under `metadata_store`

Reply:
made the change

Reply:
With that change, this section should probably be deleted?
> If you're deploying with Tanzu Application Platform profiles, in `tap-values.yaml`, put:

Reply:
oop you're right